games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="3D open-source go-kart racing game"
+DESCRIPTION="3D open-source go-kart racing game with an extensive variety of characters, tracks, and modes to play."
+HOMEPAGE="https://supertuxkart.net"
+COPYRIGHT="2020 SuperTuxKart Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/supertuxkart/stk-code/releases/download/$portVersion/supertuxkart-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="052edf0afdbeb99583fe8676fb0ab80ecb6103fb88b7540f858d1b5fa1297d37"
+SOURCE_FILENAME="supertuxkart-$portVersion-src.tar.xz"
+SOURCE_DIR="SuperTuxKart-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+ supertuxkart$secondaryArchSuffix = $portVersion
+ app:SuperTuxKart$secondaryArchSuffix
+ "
+
+REQUIRES="
+ haiku$secondaryArchSuffix
+#lib:libangelscript$secondaryArchSuffix
+ lib:libcurl$secondaryArchSuffix
+ lib:libenet$secondaryArchSuffix
+ lib:libfreetype$secondaryArchSuffix
+ lib:libGL$secondaryArchSuffix
+ lib:libglu$secondaryArchSuffix
+ lib:libglew$secondaryArchSuffix
+ lib:libharfbuzz$secondaryArchSuffix
+ lib:libjpeg$secondaryArchSuffix
+ lib:libopenal$secondaryArchSuffix
+ lib:libogg$secondaryArchSuffix
+ lib:libpng$secondaryArchSuffix
+ lib:libsdl2_2.0$secondaryArchSuffix
+ lib:libsqlite3$secondaryArchSuffix
+#lib:libsquish$secondaryArchSuffix
+ lib:libssl$secondaryArchSuffix
+ lib:libvorbis$secondaryArchSuffix
+ lib:libz$secondaryArchSuffix
+ "
+
+BUILD_REQUIRES="
+ haiku${secondaryArchSuffix}_devel
+#devel:libangelscript$secondaryArchSuffix
+ devel:libcurl$secondaryArchSuffix
+ devel:libenet$secondaryArchSuffix
+ devel:libfreetype$secondaryArchSuffix
+ devel:libGL$secondaryArchSuffix
+ devel:libglu$secondaryArchSuffix
+ devel:libglew$secondaryArchSuffix
```

Comment:
You need version >= 2 for libglew (had the devel package from the previous version that wasn't picked up without this change:
```diff
-  devel:libglew$secondaryArchSuffix
+  devel:libglew$secondaryArchSuffix >= 2
```